cli50.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+
+import argparse
+import distutils.spawn
+import os
+import pkg_resources
+import re
+import signal
+import subprocess
+import sys
+
+
+# Require python 2.7
+if sys.version_info < (2, 7):
+    sys.exit("CS50 CLI requires Python 2.7 or higher")
+if sys.version_info < (3, 0):
+    input = raw_input
+
+# Get distribution
+try:
+    __version__ = pkg_resources.get_distribution(module.__name__).version
+except:
```

Comment:
Generally considered best practice to avoid empty `except`'s. In this instance, this should probably be `except pkg_resources.DistrobutionNotFound`